packages/alfa-rules/src/common/expectation/get-aria-level.ts

```diff
@@ -0,0 +1,17 @@
+import { Node } from "@siteimprove/alfa-aria";
+import { Branched } from "@siteimprove/alfa-branched";
+import { Browser } from "@siteimprove/alfa-compatibility";
+import { Device } from "@siteimprove/alfa-device";
+import { Element } from "@siteimprove/alfa-dom";
+
+export function getAriaLevel(
+  element: Element,
+  device: Device
+): Branched<number, Browser> {
+  return Node.from(element, device).map((accNode) =>
+    accNode
+      .attribute("aria-level")
+      .map((level) => Number(level))
+      .getOr(NaN)
```

Comment:
I'd really love if the aria attribute could be typed and this conversion was not needed here…